LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2014 penzance
+Copyright (c) 2020 penzance
```

Comment:
[minor] my inclination would be to change `penzance` to `President and Fellows of Harvard College` here.

Reply:
Done! Thanks for the suggestion!